qemu/README.md

```diff
@@ -0,0 +1,154 @@
+# QEMU
+
+<p align="center">
+    <img src="logo/400x127.png" alt="QEMU">
+</p>
+
+QEMU (stands for Quick EMUlator) is an open source emulator that allows mimicking many different processors and loading many different operating systems.
+
+The Dockerfile and assets related to it are intended primarily, but **not** exclusively, for building the Docker container which simplifies testing 64-bit Raspberry Pi images built by [Pieman](https://github.com/tolstoyevsky/pieman), but not limited by this.
+
+<table>
+  <tr>
+    <td align="center" colspan="2"><b>QEMU</b></td>
+  </tr>
+  <tr>
+    <td>Version</td>
+    <td><a href="http://lists.nongnu.org/archive/html/qemu-devel/2018-04/msg04089.html">2.12.0</a></td>
+  </tr>
+  <tr>
+    <td>Release date</td>
+    <td>24 Apr 2018</td>
+  </tr>
+  <tr>
+    <td>Data volume</td>
+    <td><code>current directory:/tmp</code></td>
+  </tr>
+  <tr>
+    <td valign="top">Base images</td>
+    <td>
+        <code>cusdeb/alpine3.7:armhf</code> (for <code>armhf</code> port)<br>
+        <code>cusdeb/alpine3.7:amd64</code> (for <code>amd64</code> port)
+    </td>
+  </tr>
+</table>
+
+## Table of Contents
+
+- [Quick start](#quick-start)
+- [Documentation](#documentation)
+  * [CLI for the container](#cli-for-the-container)
+  * [The container entrypoint script](#the-container-entrypoint-script)
+- [Known issues](#known-issues)
+
+## Quick start
+
+To get started, build 64-bit custom image using Pieman or download one of the existing images from other projects (for example, you may find the 64-bit Gentoo image on [this page](https://github.com/sakaki-/gentoo-on-rpi3-64bit#gentoo-on-rpi3-64bit)).
+
+Let's say the filename of the image is `ubuntu-artful-arm64-for-rpi3.img`. To run the image in the emulator, execute the following command:
+
+```
```

Comment:
Let's add ```cd qemu``` or give a hint that this command could be running inside qemu directory of mmb, cause I got   "./docker-qemu.sh: No such file or directory" when copy-pasted command from this doc